framework/src/modules/interoperability/constants.ts

```diff
@@ -17,10 +17,18 @@ export const MODULE_NAME_INTEROPERABILITY = 'interoperability';
 
 // General constants
 export const MAINCHAIN_ID = 1;
+export const MAINCHAIN_NAME = 'lisk-mainchain';
+export const TAG_CHAIN_REG_MESSAGE = 'LSK_CHAIN_REGISTRATION';
 export const LIVENESS_LIMIT = 2592000; // 30*24*3600
 export const MAX_CCM_SIZE = 10240;
 export const EMPTY_FEE_ADDRESS = Buffer.alloc(0);
 export const EMPTY_BYTES = Buffer.alloc(0);
+export const REGISTRATION_FEE = 1000000000;
+export const MAX_NUM_VALIDATORS = 199;
+export const MAX_LENGTH_NAME = 40;
+export const MAX_UINT32 = 4294967295;
+export const MAX_UINT64 = 18446744073709551615;
```

Comment:
Can we have these values in a format like for example `2^64 – 1` for maxuint64? By the way this value is more than max number value of javascript which is 2^53-1 or 9007199254740991 so I think even though you assigned a value here, when we use this constant it will default it's value to 2^53-1 so maybe use a BigInt here or find another solution.